### PR TITLE
Change analyse_url limit to 5000

### PR DESCRIPTION
**Pull request by inglesp**
Larger than this, and the analyse form takes minutes to load.  (This is
a combination of API response time and JS running time.)  Only one
measure is affected -- the analyse_url for silver would be 37762
characters long, because the numerator is made up of so many different
BNF codes.

### CODE REVIEW

File: openprescribing/frontend/management/commands/import_measures.py [hunk 1]
```diff
@@ -358,6 +358,9 @@ def build_analyse_url(measure):
     querystring = urlencode(params)
     url = "{}#{}".format(reverse("analyse"), querystring)
 
+    if len(url) > 5000:
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/import_measures.py]: Maybe a comment inline here explaining?

Comment (inglesp) [reply openprescribing/frontend/management/commands/import_measures.py]: Good idea, added.

**Comment by inglesp** [comment]
> How fast is the next slowest? Like, less than a minute?

Yes, tens of seconds.

**Comment by inglesp** [comment]
This is now live, and here's the slowest that we allow (opioidspercent):

https://openprescribing.net/analyse/#org=CCG&numIds=0407020A0AAAFAF,0407020A0AAAGAG,0407020A0AAAHAH,0407020A0BBABAF,0407020A0BBACAG,0407020A0BBADAH,0407020A0BBAJAF,0407020A0BBAKAG,0407020A0BBALAH,0407020A0BDABAF,0407020A0BDACAG,0407020A0BDADAH,0407020A0BEACAF,0407020A0BEADAG,0407020A0BEAEAH,0407020A0BFACAF,0407020A0BFADAG,0407020A0BFAEAH,0407020A0BGABAF,0407020A0BGACAG,0407020A0BGADAH,0407020A0BHABAF,0407020A0BHACAG,0407020A0BHADAH,0407020A0BLABAF,0407020A0BLACAG,0407020A0BLADAH,0407020A0BMABAF,0407020A0BMACAG,0407020A0BMADAH,0407020A0BQACAF,0407020A0BQADAG,0407020A0BQAEAH,0407020A0BRACAG,0407020A0BRADAH,0407020A0BRAEAF,0407020A0BRAHAF,0407020A0BRAIAG,0407020A0BRAJAH,0407020A0BUACAF,0407020A0BUADAG,0407020A0BUAEAH,0407020A0BVACAF,0407020A0BVADAG,0407020A0BVAEAH,0407020ADAAAHAH,0407020ADAAAIAI,0407020ADAAAPAP,0407020ADAAAQAQ,0407020ADAAARAR,0407020ADBCACAH,0407020ADBCADAI,0407020ADBCAFAP,0407020ADBCAGAQ,0407020ADBCAHAR,0407020ADBDADAH,0407020ADBDAEAI,0407020ADBEADAH,0407020ADBEAEAI,0407020ADBEAFAP,0407020ADBEAHAR,0407020ADBEAIAQ,0407020ADBFADAH,0407020ADBFAEAI,0407020ADBKADAH,0407020ADBKAEAI,0407020ADBKAGAR,0407020ADBKAHAQ,0407020ADBLABAI,0407020ADBLACAR,0407020ADBLADAQ,0407020ADBLAEAH,0407020ADBMAEAH,0407020ADBMAFAR,0407020ADBMAGAQ,0407020ADBMAHAI,0407020ADBNAAAI,0407020ADBNABAH,0407020ADBPADAR,0407020ADBPAEAH,0407020ADBPAGAI,0407020ADBQACAH,0407020ADBQADAI,0407020ADBRAEAR,0407020ADBRAFAH,0407020ADBRAGAQ,0407020ADBSAEAR,0407020ADBSAFAH,0407020ADBSAGAQ,0407020ADBSAHAI,0407020ADBTAEAR,0407020ADBTAFAH,0407020ADBTAGAQ,0407020ADBTAHAI,0407020AFAAADAD,0407020AFBBADAD,0407020AGAAAFAF,0407020AGAAAGAG,0407020AGBBAFAF,0407020AGBBAGAG,0407020B0AAAFAF,0407020B0AAAGAG,0407020B0BDABAF,0407020B0BDACAG,0407020B0BGABAF,0407020B0BGACAG,0407020B0BIABAF,0407020B0BIACAG,0407020B0BJABAF,0407020B0BJACAG,0407020B0BNABAF,0407020B0BNACAG,0407020B0BPABAF,0407020B0BPACAG,0407020B0BSABAF,0407020B0BSACAG,0407020B0BTABAF,0407020B0BTACAG,0407020Q0AAAGAG,0407020Q0AAAHAH,0407020Q0AAAIAI,0407020Q0AADCDC,0407020Q0AADDDD,0407020Q0AADEDE,0407020Q0AADUDU,0407020Q0AADVDV,0407020Q0AADWDW,0407020Q0AAEBEB,0407020Q0AAEHEH,0407020Q0AAEIEI,0407020Q0BBACAI,0407020Q0BBADAH,0407020Q0BBAEAG,0407020Q0BBAHDC,0407020Q0BBAIDD,0407020Q0BBAJDE,0407020Q0BIABEH,0407020Q0BIACDU,0407020Q0BIADDV,0407020Q0BIAEDW,0407020Q0BIAFEI,0407020Q0BJ,0407020Q0BKACEH,0407020Q0BKADEB,0407020Q0BKAEEI,0407020Q0BMACAI,0407020Q0BMADAH,0407020Q0BMAEAG,0407020Q0BNACAI,0407020Q0BNADAH&denomIds=0407020A0AAAEAE,0407020A0AAAFAF,0407020A0AAAGAG,0407020A0AAAHAH,0407020A0AAAUAU,0407020A0AABWBW,0407020A0BBAAAE,0407020A0BBABAF,0407020A0BBACAG,0407020A0BBADAH,0407020A0BBAIAE,0407020A0BBAJAF,0407020A0BBAKAG,0407020A0BBALAH,0407020A0BBAMAU,0407020A0BD,0407020A0BE,0407020A0BF,0407020A0BG,0407020A0BH,0407020A0BL,0407020A0BM,0407020A0BQ,0407020A0BR,0407020A0BU,0407020A0BV,0407020ADAAAFAF,0407020ADAAAGAG,0407020ADAAAHAH,0407020ADAAAIAI,0407020ADAAAKAK,0407020ADAAAPAP,0407020ADAAAQAQ,0407020ADAAARAR,0407020ADAAASAS,0407020ADBC,0407020ADBD,0407020ADBE,0407020ADBF,0407020ADBK,0407020ADBL,0407020ADBM,0407020ADBN,0407020ADBP,0407020ADBQ,0407020ADBR,0407020ADBS,0407020ADBT,0407020AFAAAAAA,0407020AFAAABAB,0407020AFAAACAC,0407020AFAAADAD,0407020AFBB,0407020AGAAACAC,0407020AGAAADAD,0407020AGAAAEAE,0407020AGAAAFAF,0407020AGAAAGAG,0407020AGBBACAC,0407020AGBBADAD,0407020AGBBAEAE,0407020AGBBAFAF,0407020AGBBAGAG,0407020B0AAAEAE,0407020B0AAAFAF,0407020B0AAAGAG,0407020B0AAAHAH,0407020B0AAAIAI,0407020B0AAAJAJ,0407020B0BD,0407020B0BEAAAH,0407020B0BEABAI,0407020B0BEACAJ,0407020B0BG,0407020B0BHAAAH,0407020B0BHABAI,0407020B0BHACAJ,0407020B0BI,0407020B0BJ,0407020B0BKAAAH,0407020B0BKABAI,0407020B0BKADAJ,0407020B0BL,0407020B0BM,0407020B0BN,0407020B0BP,0407020B0BQ,0407020B0BR,0407020B0BS,0407020B0BT,0407020B0BU,0407020Q0AAAGAG,0407020Q0AAAHAH,0407020Q0AAAIAI,0407020Q0AAAKAK,0407020Q0AAALAL,0407020Q0AACFCF,0407020Q0AACGCG,0407020Q0AACPCP,0407020Q0AACVCV,0407020Q0AADCDC,0407020Q0AADDDD,0407020Q0AADEDE,0407020Q0AADUDU,0407020Q0AADVDV,0407020Q0AADWDW,0407020Q0AADZDZ,0407020Q0AAEBEB,0407020Q0AAEFEF,0407020Q0AAEGEG,0407020Q0AAEHEH,0407020Q0AAEIEI,0407020Q0BB,0407020Q0BI,0407020Q0BJ,0407020Q0BK,0407020Q0BM,0407020Q0BN&selectedTab=summary

**Pull request merged**